package-lock.json

```diff
@@ -13848,6 +13844,69 @@
         }
       }
     },
+    "postcss-prefix-selector": {
```

Comment:
The package-lock changes are more extensive than just this new package. Can you revert the package-lock.json changes, except for this new piece?

Reply:
I'll take a look and revert.